packages/@storefront/sayt/test/unit/autocomplete.ts

```diff
@@ -356,27 +385,54 @@ suite('Autocomplete', ({ expect, spy, stub, itShouldProvideAlias }) => {
       expect(autocomplete.updateProducts).to.not.have.been.called;
     });
 
-    it('should call update products with updateQuery false', () => {
+    it('should call parseTarget', () => {
       const index = 0;
-      const state = (autocomplete.state = <any>{ selected: 4 });
+      const target = { classList: { add: () => null } };
+      const parseTarget = (autocomplete.parseTarget = spy());
+      autocomplete.updateProducts = () => null;
+
+      autocomplete.setActivation(<any>[target], index, true, false);
+
+      expect(parseTarget).to.be.calledWithExactly(target);
+    });
+
+    it('should call updateQuery with the query string', () => {
+      const index = 0;
+      const target = { classList: { add: () => null } };
+      const query = 'foo';
+      const updateQuery = (autocomplete.updateQuery = spy());
+      autocomplete.parseTarget = spy(() => ({ query }));
```

Comment:
Doesn't need to be a `spy` if there are no assertions on it.

Reply:
Good call. Will update.

Reply:
Addressed. Resolving.